src/runtime/contrib/tensorrt/tensorrt_builder.cc

```diff
@@ -208,8 +208,22 @@ runtime::TrtEngineAndContext TensorRTBuilder::BuildEngine(
   CHECK_EQ(engine->getNbBindings(),
            num_input_bindings + network_output_names_.size());
   nvinfer1::IExecutionContext* context = engine->createExecutionContext();
-  return {engine, context, network_input_names_, network_input_is_baked_,
-          network_output_names_};
+  std::vector<runtime::NDArray> device_buffers;
+  for (int i = 0; i < execution_args_.size(); i++) {
+    if (execution_args_[i]->ctx.device_type == kDLGPU) {
+      device_buffers.push_back(runtime::NDArray());
+    } else {
+      std::vector<int64_t> shape_;
```

Comment:
Let use `std::vector<int64_t> shape(execution_args_[i]->shape, execution_args_[i]->shape + execution_args_[i]->ndim)` instead of the loop.

Reply:
Done,